var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -241,6 +243,16 @@ def cmake_args(self):
                 '-DDEAL_II_WITH_OPENCASCADE=OFF'
             ])
 
+        # 64 bit indices
+        if '+64bit' in spec:
+            options.extend([
+                '-DDEAL_II_WITH_64BIT_INDICES=ON'
```

Comment:
I don't mind merging as-is but this could probably shorten up some peoples' code.